lib/components/SelectionControl/radio/radio-group/radio-group.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import RadioGroup from "./index";
+import { fn } from "@storybook/test";
+
+interface RadioGroupProps {
+    disabledOption?: boolean;
+    hiddenOption?: boolean;
+    name?: string;
+    onToggle?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    selected?: string;
+    shouldWrapItems?: boolean;
+}
+
+const RadioGroupItem = ({
+    value = "",
+    label = "",
+    disabled = false,
+    hidden = false,
```

Comment:
do you think it will be beneficial to add all these default values to props of `RadioGroup.Item`? :) this way you till not need to have this `RadioGroupItem` wrapper

Reply:
This is just here because of typescript gymnastics, some of these values can't be assigned directly to a div element as it's not part of HTMLElement type, so I added the default values here to get rid of typescript complains. I don't think there are any benefits.